src/transport/SessionManager.h

```diff
@@ -168,6 +168,21 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     CHIP_ERROR NewPairing(SessionHolder & sessionHolder, const Optional<Transport::PeerAddress> & peerAddr, NodeId peerNodeId,
                           PairingSession * pairing, CryptoContext::SessionRole direction, FabricIndex fabric);
 
+    /**
+     * @brief
+     *   Allocate a secure session and non-colliding session ID in the secure
+     *   session table.
+     */
+    SessionHolder AllocateSession();
+
+    /**
+     * @brief
+     *   Allocate a secure session in the secure session table at the specified
+     *   session ID.  If the session ID collides with an existing session, evict
+     *   it.
```

Comment:
Again, if this is mostly meant for tests we should document that.

Reply:
Done.